apps/linear_algebra/Makefile

```diff
@@ -7,8 +7,8 @@ EMIT_OPTIONS = stmt,assembly
 
 EIGEN_INCLUDES ?= -I/usr/include/eigen3
 CBLAS_LIBS ?= -lblas
-ATLAS_FLAGS ?= -DUSE_ATLAS
-ATLAS_LIBS ?= -L/usr/lib/atlas-base -lcblas -latlas
+ATLAS_FLAGS ?= -DUSE_ATLAS -I/opt/ATLAS/include
+ATLAS_LIBS ?= -L/opt/ATLAS/lib -lptcblas -latlas
```

Comment:
You installed atlas locally to get the pthreads version? It might be better if the Makefile assumed the standard system install, and people with a custom install can do make ATLAS_LIBS=...